### PR TITLE
ci: upgrade to yarn 3

### LINKED ISSUE #21: Upgrade to yarn 3

### PR DESCRIPTION
**Pull request by pyphilia**
close #21

### CODE REVIEW

File: .husky/commit-msg [hunk 1]
```diff
@@ -2,4 +2,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn commit-msg
+npx --no-install commitlint --edit "$1"
```
Comment (juancarlosfarah) [inline .husky/commit-msg:5]: Why do we need to run this now with `npx`?

File: package.json [hunk 2]
```diff
@@ -68,8 +67,9 @@
     ]
   },
   "devDependencies": {
-    "@commitlint/cli": "12.1.1",
-    "@commitlint/config-conventional": "12.1.1",
+    "@babel/eslint-parser": "^7.15.0",
+    "@commitlint/cli": "^13.1.0",
+    "@commitlint/config-conventional": "^13.1.0",
```
Comment (juancarlosfarah) [inline package.json:72]: Still not sure why these can't be pinned. Maybe we can discuss this on Slack?

File: package.json [hunk 3]
```diff
@@ -98,5 +101,6 @@
     "exclude": [
       "src/serviceWorker.js"
     ]
-  }
+  },
+  "packageManager": "yarn@3.0.1-rc.1"
```
Comment (juancarlosfarah) [inline package.json:105]: Does this mean that every time we want to upgrade the package manager we need to edit this line and change the file in `.yarn/releases`? Or is it done automatically?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good! I just have some questions. Maybe we can discuss them on Slack. Thanks!

File: package.json (hunk 3 above)
Comment (pyphilia) [reply package.json:105]: Everything is automatic!

File: .husky/commit-msg (hunk 1 above)
Comment (pyphilia) [reply .husky/commit-msg:5]: I am honestly not really sure, but our previous setup was failing. And they do exactly the following in the official docs https://github.com/conventional-changelog/commitlint So it's also nice for us as it removes a hook script from `package.json`.

**Pull request merged**